frontend/src/metabase/query_builder/components/DatasetEditor/ResizableNotebook/ResizableNotebook.jsx

```diff
@@ -29,7 +29,7 @@ const Handle = forwardRef(function Handle(props, ref) {
       w="100%"
       h="sm"
       pos="absolute"
-      bottom="-0.25rem"
+      bottom={`-${rem(4)}`}
```

Comment:
BTW `rem(-4)` should work

```suggestion
      bottom={rem(-4)}
```

Reply:
Good thinking.
Thanks